index.d.ts

```diff
@@ -32,4 +32,4 @@ await waitForLocalhost({port: 8080});
 console.log('Server is ready');
 ```
 */
-export default function waitForLocalhost(options?: Options): Promise<void>;
+export default function waitForLocalhost(options?: Options): Promise<{family: 4|6}>;
```

Comment:
I think it should be called `ipVersion`, not `family`. That's clearer to users.

Reply:
Yes, makes sense. I just followed what node did.